docs/Manual/09 Parameters.md

```diff
@@ -32,10 +32,10 @@ package = acc.Package()
 package.add(nest, args=(A, B, C), parameters={P0:16, P1:16, P2:16, P3:1.0}, base_name="matmul_16_16_16_1")
 package.add(nest, args=(A, B, C), parameters={P0:32, P1:32, P2:32, P3:2.0}, base_name="matmul_32_32_32_2")
 ```
-In this example, the shape of the nest is parameterized by (`P0`, `P1`, `P2`) and its iteration logic includes the parameter `P3`. The nest is used twice, with different settings of these parameters, to create two separate functions in the package.
+In the above scenario, the shape of the nest is parameterized by (`P0`, `P1`, `P2`) and its iteration logic includes the parameter `P3`. The nest is used twice with different settings of these parameters to create two separate functions in the package.
 
 ## Parameterized schedules and plans
-Parameters can also appear in schedules and plans. For example, we could add the following to the code above:
+As mentioned before, parameters can be used in schedules and plans. For example, we can add the following code snippet to the above logic:
```

Comment:
This should be the first time we discuss parameters in schedules and plans, because the previous section covers parameters in nests. 

So: "Parameters can also appear in schedules and plans." flows better.

Reply:
Ah, I see. Actually I meant, in the lines above.

Reply:
Addressed.